src/manager.js

```diff
@@ -417,16 +432,16 @@ AdManager.prototype.loadAds = function(element, updateCorrelator) {
  * @returns undefined
 */
 AdManager.prototype.refreshSlot = function(domElement) {
-  if ((domElement.getAttribute('data-ad-load-state') === 'loaded') || (domElement.getAttribute('data-ad-load-state') === 'loading')) {
-    return;
+  var that = this;
+  if (this.options.amazon_enabled) {
```

Comment:
not :100: on how do go about testing this conditional, any thoughts @spra85 @collin @kand @brycedorn?